website/addons/googledrive/utils.py

```diff
@@ -27,6 +28,7 @@ def to_hgrid(item, node, path):
         'kind': 'folder',
         'name': safe_name,
         'addon': 'googledrive',
+        'provider': 'googledrive',  # TODO: remove duplication -- APIv2
```

Comment:
I see. I assume this is because Google Drive specifically was inconsistent, while the other addons already contained a key named `provider`?

(if every provider has the same apiv1/apiv2 difference, an alternative would be to specify the `source=` argument in the DRF serializer field definition. Different ways of approaching the duplicate code problem)